components/rmrk/Gallery/Gallery.vue

```diff
@@ -305,7 +305,8 @@ export default class Gallery extends mixins(PrefixMixin) {
 
 .gallery {
   @media screen and (max-width: 1023px) {
-    padding: 0 15px;
+    padding: 0;
+    margin: 0 -0.5rem;
   }
```

Comment:
I would have removed this to align content with burger-menu

Reply:
negative margin bad!

Reply:
Yes but the other option was to remain padding from the `section` class which is applied on all the pages, then I would have to apply padding manually on all the other pages @roiLeo

Reply:
#### current
![Screenshot 2022-01-02 at 18-22-20 KodaDot - Polkadot Kusama NFT explorer](https://user-images.githubusercontent.com/9987732/147883921-cbcffe3c-0904-4499-9b6c-37d46965da43.png)


#### your solution
![Screenshot 2022-01-02 at 18-22-03 nft-gallery-nuxt](https://user-images.githubusercontent.com/9987732/147883927-88950ec9-82c2-49fc-8a41-7c5edc360b50.png)


#### without padding
![Screenshot 2022-01-02 at 18-21-24 nft-gallery-nuxt](https://user-images.githubusercontent.com/9987732/147883938-cb4583fa-d612-4ded-aaf1-874fe691a72f.png)

Reply:
I don't know why these lines had been introduced only on these component